pkg/compose/push.go

```diff
@@ -139,10 +139,12 @@ func toPushProgressEvent(prefix string, jm jsonmessage.JSONMessage, w progress.W
 		return
 	}
 	var (
-		text   string
-		status = progress.Working
+		text    string
+		status  = progress.Working
+		total   int64
+		current int64
 	)
-	if jm.Status == "Pull complete" || jm.Status == "Already exists" {
+	if jm.Status == "Pushed" || jm.Status == "Already exists" {
```

Comment:
can this strings not use some sort of enum or const ?
(I don't know golang)

Reply:
unfortunately golang doesn't have enums
ideally, const should be defined by the Docker Engine API